crates/bevy_render/src/view/window.rs

```diff
@@ -227,8 +227,12 @@ pub fn prepare_windows(
                 CompositeAlphaMode::PostMultiplied => wgpu::CompositeAlphaMode::PostMultiplied,
                 CompositeAlphaMode::Inherit => wgpu::CompositeAlphaMode::Inherit,
             },
-            // TODO: Use an sRGB view format here on platforms that don't support sRGB surfaces. (afaik only WebGPU)
-            view_formats: vec![],
+            // TODO: replace by `is_srgb()` once wgpu 0.16 is released
+            view_formats: if surface_data.format != surface_data.format.remove_srgb_suffix() {
```

Comment:
Could you flip the logic here and have == and that case first? It would be easier to read.